### PR TITLE
[i18n-ar] Translated file : `docs/source/ar/community.md` into Arabic

### PR DESCRIPTION
**Pull request by AhmedAlmaghz**
## What does this PR do?
Translated the `docs/source/ar/community.md` file of the documentation to `Arabic العربية`. I use transformers for my side-project and wanted to contribute. As a beginner, this seemed appropriate.

### Some questions:

- I can't access the pr documentation endpoint for `Arabic العربية`. Is there a missing step?
- I left all the other files alone after copying the English docs. Is this acceptable for future updates or should I add contents to the `_toctree.yml` only when translation is complete? Should I delete all other files for a cleaner review?
- In `Arabic`, people use both `Transformer` and `المحولات` to describe the model. However, as this is more of a product of HuggingFace I opted not to translate it. May you please let me know which you would prefer?

Thank you in advance for your review.

Part of #32435

### Before submitting

- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests), Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/) ?Please add a link to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

 ### Who can review?
 @stevhliu , @MKhalusova , @abodacs, may you please review this PR?

### CODE REVIEW

**Comment by AhmedAlmaghz** [comment]
أخي عبدالله @abodacs  هذا ما تم إنجازه في هذا القسم 
إلى الامام إن شاء الله
هذا آخر ملف في هذا القسم



# DEVELOPER GUIDES

- [x]  Use fast tokenizers from 🤗 Tokenizers #33034
- [x]  Run inference with multilingual models #33048
- [x]  Use model-specific APIs #33030
- [x]  Share a custom model #33031
- [x]  Templates for chat models #33026
- [x]  Trainer #33080
- [x]  Run training on Amazon SageMaker #33071
- [x]  Export to ONNX #33072
- [x]  Export to TFLite #33077
- [ ]  Export to TorchScript #33079
- [ ]  Benchmarks #33023
- [ ]  Notebooks with examples #33049
- [ ]  Community resources #33027
- [ ]  Troubleshoot #33017
- [x]  Interoperability with GGUF files #33037

**Comment by abodacs** [comment]
@AhmedAlmaghz 
تسلم
 ممكن تلقى نظرة على
 
https://huggingface.co/docs/transformers/main/en/tiktoken
https://huggingface.co/docs/transformers/main/en/modular_transformers
https://huggingface.co/docs/transformers/main/en/how_to_hack_models

اعتقد انهم ليس لهم ترجمة

**Review by abodacs — APPROVED** [review]

**Comment by AhmedAlmaghz** [comment]
> @AhmedAlmaghz تسلم ممكن تلقى نظرة على
> 
> https://huggingface.co/docs/transformers/main/en/tiktoken https://huggingface.co/docs/transformers/main/en/modular_transformers https://huggingface.co/docs/transformers/main/en/how_to_hack_models
> 
> اعتقد انهم ليس لهم ترجمة

تم إضافة ترجمة الملفات 
هذه الملفات جديدة تم إضافتها حديثاً

اشكري اخي عبدالله @abodacs على ملاحظاتك  القيمة.

اتمنى لك التوفيق

File: docs/source/ar/modular_transformers.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+[TODO] نقدم اختبارًا جديدًا، للتأكد من أن المحتوى المولد يتطابق مع ما هو موجود في `modular_xxxx.py`
+
+### الأمثلة
+
+هنا مثال سريع مع BERT و RoBERTa. النموذجان مرتبطان ارتباطًا وثيقًا: يختلف تنفيذهما النموذجي
+في طبقة تضمين.
+
+بدلاً من إعادة تعريف النموذج بالكامل، إليك كيف يبدو ملف `modular_roberta.py` لفئات النمذجة والتكوين (لأغراض المثال، يتم تجاهل المحلل اللغوي في هذا الوقت حيث أنه مختلف جدًا).
+
```
Comment (abodacs) [inline docs/source/ar/modular_transformers.md:67]:
```suggestion
هنا مثال سريع باستخدام BERT و RoBERTa. النموذجان مرتبطان ارتباطًا وثيقًا: يختلف تنفيذهما النموذجي في طبقة تضمين.

بدلاً من إعادة تعريف النموذج بالكامل، إليك كيف يبدو ملف `modular_roberta.py` لفئات النمذجة والتكوين (لأغراض المثال، يتم تجاهل المجزىء اللغوي في هذا الوقت حيث أنه مختلف جدًا).

```

File: docs/source/ar/tiktoken.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+يتم دمج دعم ملفات نموذج tiktoken بسلاسة في 🤗 transformers عند تحميل النماذج
+`from_pretrained` مع ملف `tokenizer.model` tiktoken على Hub، والذي يتم تحويله تلقائيًا إلى [المحلل اللغوي السريع](https://huggingface.co/docs/transformers/main/en/main_classes/tokenizer#transformers.PreTrainedTokenizerFast).
+
+### النماذج المعروفة التي تم إصدارها مع `tiktoken.model`:
+	- gpt2
+	- llama3
+
+## مثال على الاستخدام
+
+من أجل تحميل ملفات `tiktoken` في `transformers`، تأكد من أن ملف `tokenizer.model` هو ملف tiktoken وسيتم تحميله تلقائيًا عند التحميل `from_pretrained`. إليك كيفية تحميل محلل لغوي ونموذج، والذي
```
Comment (abodacs) [inline docs/source/ar/tiktoken.md:12]:
```suggestion
من أجل تحميل ملفات `tiktoken` في `transformers`، تأكد من أن ملف `tokenizer.model` هو ملف tiktoken وسيتم تحميله تلقائيًا عند التحميل `from_pretrained`. إليك كيفية تحميل مجزىء لغوي ونموذج، والذي
```

File: docs/source/ar/tiktoken.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+يمكن تحميله من نفس الملف بالضبط:
+
+```py
+from transformers import AutoTokenizer
+
+model_id = "meta-llama/Meta-Llama-3-8B-Instruct"
+tokenizer = AutoTokenizer.from_pretrained(model_id, subfolder="original")
+```
+## إنشاء محلل لغوي tiktoken
```
Comment (abodacs) [inline docs/source/ar/tiktoken.md:21]:
```suggestion
## إنشاء مجزىء لغوي tiktoken
```

**Review by abodacs — CHANGES_REQUESTED** [review]
السﻻم عليكم

تمت المراجعة جزاك الله خير

@AhmedAlmaghz

**Comment by AhmedAlmaghz** [comment]
Thank you @abodacs

**Review by stevhliu — APPROVED** [review]
LGTM, thanks!

**Pull request merged**